packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -492,7 +492,7 @@ Terra.describeViewports('ModalManager - Disclosure Accessory', ['large'], () =>
 
     it('renders each disclosed component in a disclosure container', () => {
       expect(browser.isExisting('[data-disclosure-container="true"]')).to.be.true;
-      Terra.validates.element('nested disclosure', { selector });
+      Terra.validates.screenshot('nested disclosure', { selector });
```

Comment:
Can you remove the `only` on this describe block? It was something I accidently merged in

Reply:
removed here https://github.com/cerner/terra-framework/pull/989/commits/dc3742c06e6b79e7055729172d004837736b891a